doc/cephadm/monitoring.rst

```diff
@@ -77,8 +77,20 @@ monitoring by following the steps below.
 
      ceph orch apply grafana 1
 
-Cephadm handles the prometheus, grafana, and alertmanager
-configurations automatically.
+Cephadm takes care of the configuration of Prometheus, Grafana, and Alertmanager
+automatically.
+
+However, there is one exception to this rule. In a some setups, the Dashboard
+user's browser might not be able to access the URL to Grafana which is
```

Comment:
This is somewhat difficult to parse in English. This would be simpler:

`s/the URL to Grafana which is configured/the Grafana URL configured/`